sdk/storage/sample.env

```diff
@@ -1,3 +1,4 @@
 ACCOUNT_NAME=<storage-account-name>
 ACCOUNT_KEY=<storage-account-key>
-ACCOUNT_SAS=<shared-access-signature>
\ No newline at end of file
+ACCOUNT_SAS=<shared-access-signature>
+STORAGE_CONNECTION_STRING=<storage-account-connection-string>
```

Comment:
Where does this get referred?

Reply:
`sample.env` is just a file to show how the environment(`.env` file) should look like.

`STORAGE_CONNECTION_STRING` environment variable is being used in some of the tests where we test the connection string static methods and constructors. [Unrelated to this PR]